stripe/__init__.py

```diff
@@ -11,12 +11,13 @@
 # Andrew Metcalf <andrew@stripe.com>
 
 # Configuration variables
-from stripe.api_version import _ApiVersion
+from stripe._api_version import _ApiVersion
 
-from stripe.app_info import AppInfo
-
-if TYPE_CHECKING:
-    from stripe.http_client import HTTPClient
+# We must import the app_info module eagerly before defining the app_info global
+# otherwise the late import will overwrite the global
+import stripe.app_info
```

Comment:
Wouldn't this overwriting behavior have been true before?

Reply:
No, because we were importing from app_info before so the app_info modules was already loaded before `app_info = None` line runs.